src/libsystemd/sd-journal/journal-verify.c

```diff
@@ -280,7 +280,7 @@ static int journal_file_object_verify(JournalFile *f, uint64_t offset, Object *o
                         if (le64toh(o->entry.items[i].object_offset) == 0 ||
                             !VALID64(le64toh(o->entry.items[i].object_offset))) {
                                 error(offset,
-                                      "Invalid entry item (%"PRIu64"/%"PRIu64" offset: "OFSfmt,
+                                      "Invalid entry item (%"PRIu64"/%"PRIu64" offset: "OFSfmt")",
```

Comment:
Should be like this for consistency:
```suggestion
                                      "Invalid entry item (%"PRIu64"/%"PRIu64") offset: "OFSfmt,
```